contrib/debian/fwupd-qubes-vm-whonix.postinst

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+FWUPD_UPDATEVM_DIR=/home/user/.cache/fwupd
```

Comment:
`user` is hardcoded to a fixed name?  Doesn't there need to be some sort of lookup process for the correct ID and group?

Reply:
As there is a possibility that someone generates a build with this included and tries to install that `.deb` package, can you please add directory guards to make sure that `/home/user` exists and exit the `postinst` otherwise?

Reply:
Another (better) safety check would be to do a check using:
`getent passwd user`

You can check both the return code as well as the home directory assigned to the user so you can avoid hardcoding anything. I f the return code is non-zero don't run anything else.

Reply:
I've added safety check.